packages/aws-otel-collector/aws-otel-collector.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=AWS OTEL collector
+After=network.target
```

Comment:
```suggestion
After=network-online.target configured.target
Wants=network-online.target configured.target
```

Reply:
Ack, from what I've read online, the reason we may want to do this is to ensure the network is up before we start the OTEL collector service. Is this correct, or is there another reason for this suggestion?

Reply:
I would use network-online.target in any unit file that's going to do network-stuff (including otel, of course). It's the target that says "hooray, we have an interface and it is configured and you can actually do TCP and DNS stuff on it, good luck, have fun." So for instance, this is a prerequisite for IMDS, and makes sense as a prerequisite for otel collector.